reaktive/api/reaktive.api

```diff
@@ -57,7 +57,7 @@ public final class com/badoo/reaktive/base/exceptions/TimeoutException : java/la
 }
 
 public final class com/badoo/reaktive/completable/AmbKt {
-	public static final fun amb (Ljava/lang/Iterable;)Lcom/badoo/reaktive/completable/Completable;
+	public static final fun amb (Ljava/util/Collection;)Lcom/badoo/reaktive/completable/Completable;
```

Comment:
@arkivanov 

Binary compatibility can be possibly broken. Not all Iterables are Collections.
But maybe it is fine to change it like this, not sure if anybody passes here pure custom Iterable implementation.